tests/fast-integration/skr-svcat-migration-test/test-helpers.js

```diff
@@ -0,0 +1,85 @@
+const installer = require("../installer/helm");
+const k8s = require("@kubernetes/client-node");
+const execa = require("execa");
+
+async function installHelmCharts(creds) {
+  const btpChart = "https://github.com/kyma-incubator/sap-btp-service-operator/releases/download/v0.1.9-custom/sap-btp-operator-custom-0.1.9.tar.gz";
+  const btp = "sap-btp-operator";
+  const btpValues = `manager.secret.clientid=${creds.clientId},manager.secret.clientsecret=${creds.clientSecret},manager.secret.url=${creds.smURL},manager.secret.tokenurl=${creds.url},cluster.id=${creds.clusterId}`
+  try {
+    await installer.helmInstallUpgrade(btp, btpChart, btp, btpValues, null, ["--create-namespace"]);
+  } catch(error) {
+    if (error.stderr === undefined) {
+      throw new Error(`failed to install btp-operator: ${error}`);
+    }
+    throw new Error(`failed to install btp-operator: ${error.stderr}`);
+  }
+}
+
+async function smInstanceBinding(url, clientid, clientsecret, svcatPlatform, btpOperatorInstance, btpOperatorBinding) {
+  try {
+    let args = [`login`, `-a`, url, `--param`, `subdomain=e2etestingscmigration`, `--auth-flow`, `client-credentials`]
+    await execa(`smctl`, args.concat([`--client-id`, clientid, `--client-secret`, clientsecret]));
+
+    args = [`register-platform`, svcatPlatform, `kubernetes`]
+    await execa(`smctl`, args);
+
+    args = [`provision`, btpOperatorInstance, `service-manager`, `service-operator-access`, `--mode=sync`]
+    await execa(`smctl`, args);
+
+    args = [`bind`, btpOperatorInstance, btpOperatorBinding, `--mode=sync`];
+    await execa(`smctl`, args);
+    args = [`get-binding`, btpOperatorBinding, `-o`, `json`];
+    let out = await execa(`smctl`, args);
+    let b = JSON.parse(out.stdout)
+    let c = b.items[0].credentials
+    //TODO figure out how to find clusterid
```

Comment:
```
smctl list-instances -o json | jq '.items[] | select(.name == "uaa-issuer").context.clusterid'
```
this is what I used in the SKRs I owned, but for e2es the `uaa-issuer` is not present.